resources/config.yml

```diff
@@ -58,13 +58,13 @@ Give-Experience-For-Feeding: true
 # Options:
 # - MySQL
 # - SQLite
-Database: SQLite
+Database: MySQL
```

Comment:
As much as I'd prefer MySQL to be the default database format, it requires a manual setup. SQLite3 on the other hand is pretty straightforward and doesn't require any setup.